ecs/image_config.go

```diff
@@ -25,17 +25,18 @@ type AlicloudDiskDevices struct {
 }
 
 type AlicloudImageConfig struct {
-	AlicloudImageName                 string   `mapstructure:"image_name"`
-	AlicloudImageVersion              string   `mapstructure:"image_version"`
-	AlicloudImageDescription          string   `mapstructure:"image_description"`
-	AlicloudImageShareAccounts        []string `mapstructure:"image_share_account"`
-	AlicloudImageUNShareAccounts      []string `mapstructure:"image_unshare_account"`
-	AlicloudImageDestinationRegions   []string `mapstructure:"image_copy_regions"`
-	AlicloudImageDestinationNames     []string `mapstructure:"image_copy_names"`
-	AlicloudImageForceDelete          bool     `mapstructure:"image_force_delete"`
-	AlicloudImageForceDeleteSnapshots bool     `mapstructure:"image_force_delete_snapshots"`
-	AlicloudImageForceDeleteInstances bool     `mapstructure:"image_force_delete_instances"`
-	AlicloudImageSkipRegionValidation bool     `mapstructure:"skip_region_validation"`
+	AlicloudImageName                 string            `mapstructure:"image_name"`
+	AlicloudImageVersion              string            `mapstructure:"image_version"`
+	AlicloudImageDescription          string            `mapstructure:"image_description"`
+	AlicloudImageShareAccounts        []string          `mapstructure:"image_share_account"`
+	AlicloudImageUNShareAccounts      []string          `mapstructure:"image_unshare_account"`
+	AlicloudImageDestinationRegions   []string          `mapstructure:"image_copy_regions"`
+	AlicloudImageDestinationNames     []string          `mapstructure:"image_copy_names"`
+	AlicloudImageForceDelete          bool              `mapstructure:"image_force_delete"`
+	AlicloudImageForceDeleteSnapshots bool              `mapstructure:"image_force_delete_snapshots"`
+	AlicloudImageForceDeleteInstances bool              `mapstructure:"image_force_delete_instances"`
+	AlicloudImageSkipRegionValidation bool              `mapstructure:"skip_region_validation"`
+	AlicloudImageTags                 map[string]string `mapstructure:"tags"`
```

Comment:
I think `AlicloudImageTags`'s type should be `map[string]interface{}`.

Reply:
Tags are key/value pairs and only `string` is supported in tags